directFeatures.py

```diff
@@ -69,16 +72,13 @@ def match_to_faces(list_face_encodings, list_face_locations, people, resized_ima
         list_face_names.append(name)
 
         for (top, right, bottom, left), name in zip(list_face_locations, list_face_names):
-            cv2.rectangle(resized_image, (left, top),
-                          (right, bottom), (255, 0, 0), 2)
+            cv2.rectangle(resized_image, (left-5, top-5),
+                          (right+5, bottom+5), (255, 0, 0), 2)
```

Comment:
based on interpreter testing, cv2 accepts out of bounds coordinates without error and draws partial box as desired